docs/api.md

```diff
@@ -3623,50 +3635,24 @@ const backgroundPage = await backroundPageTarget.page();
 - [browserContext.setPermissions(origin, permissions[])](#browsercontextsetpermissionsorigin-permissions)
 <!-- GEN:stop -->
 
-#### event: 'targetchanged'
-- <[ChromiumTarget]>
-
-Emitted when the url of a target changes.
-
-> **NOTE** Only includes targets from this browser context.
-
-
-#### event: 'targetcreated'
-- <[ChromiumTarget]>
+#### event: 'backgroundpage'
+- <[Page]>
 
-Emitted when a target is created, for example when a new page is opened by [`window.open`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) or [`browserContext.newPage`](#browsercontextnewpage).
+Emitted when new background page is created in the context.
 
-> **NOTE** Only includes targets from this browser context.
+> **NOTE** Only works with persistent context.
```

Comment:
Any chance you could provide more context on why this only works in persistent context?

Reply:
It looks like the background pages are always created in the default context which doesn't work well with browser context level isolation that we want to have. If this becomes a problem we can extend the API in the future.